### ISSUE TITLE
Could not load dynamic library 'libcudart.so.11.0'

### ISSUE THREAD

**Issue by thegregyang** [google/jax#3746]
I installed jax via the automatic command in the README

```
pip install --upgrade https://storage.googleapis.com/jax-releases/`nvidia-smi | sed -En "s/.* CUDA Version: ([0-9]*)\.([0-9]*).*/cuda\1\2/p"`/jaxlib-0.1.51-`python3 -V | sed -En "s/Python ([0-9]*)\.([0-9]*).*/cp\1\2/p"`-none-manylinux2010_x86_64.whl jax
```

When I run `python -c 'import jax.numpy as np; print(np.zeros([3, 4]))'` I get

```
2020-07-13 23:14:21.441528: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2020-07-13 23:14:21.441564: I external/org_tensorflow/tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2020-07-13 23:14:21.496885: W external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2020-07-13 23:14:21.496916: I external/org_tensorflow/tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2020-07-13 23:14:21.735723: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564f52e8b120 initialized for platform Interpreter (this does not guarantee that XLA will be used). Devices:
2020-07-13 23:14:21.735762: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Interpreter, <undefined>
2020-07-13 23:14:21.739108: I external/org_tensorflow/tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2593990000 Hz
2020-07-13 23:14:21.740110: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564f52e881c0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-07-13 23:14:21.740202: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-07-13 23:14:21.744840: I external/org_tensorflow/tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-07-13 23:14:21.994105: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564f52b71da0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-07-13 23:14:21.994151: I external/org_tensorflow/tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Tesla P100-PCIE-16GB, Compute Capability 6.0
2020-07-13 23:14:21.994593: I external/org_tensorflow/tensorflow/compiler/xla/pjrt/nvidia_gpu_device.cc:109] XLA backend allocating 15121907712 bytes on device 0 for BFCAllocator.
2020-07-13 23:14:22.273783: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:70] Can't find libdevice directory ${CUDA_DIR}/nvvm/libdevice. This may result in compilation or runtime failures, if the program we try to run uses routines from libdevice.
2020-07-13 23:14:22.273824: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:71] Searched for CUDA in the following directories:
2020-07-13 23:14:22.273839: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:74]   ./cuda_sdk_lib
2020-07-13 23:14:22.273849: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:74]   /usr/local/cuda-11.0
2020-07-13 23:14:22.273859: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:74]   .
2020-07-13 23:14:22.273869: W external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:76] You can choose the search directory by setting xla_gpu_cuda_data_dir in HloModule's DebugOptions.  For most apps, setting the environment variable XLA_FLAGS=--xla_gpu_cuda_data_dir=/path/to/cuda will work.
2020-07-13 23:14:22.303006: W external/org_tensorflow/tensorflow/stream_executor/gpu/asm_compiler.cc:81] Running ptxas --version returned 256
2020-07-13 23:14:22.322283: F external/org_tensorflow/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:419] ptxas returned an error during compilation of ptx to sass: 'Internal: ptxas exited with non-zero error code 256, output: '  If the error message indicates that a file could not be written, please verify that sufficient filesystem space is provided.
Aborted (core dumped)
```

This is what I get when I run `nvidia-smi`

![image](https://user-images.githubusercontent.com/53244851/87364334-0bcccb80-c539-11ea-891e-e51f4a33fda6.png)

**Comment by hawkinsp** [comment]
Where is `libcudart.so.11.0` installed on your machine? JAX is most likely looking in `/usr/local/cuda`.

**Comment by skye** [comment]
Also see the section in https://github.com/google/jax#installation above the automatic installation command, it has some info for how to tell JAX where cuda is installed.

**Comment by hawkinsp** [comment]
Closing because there has been no response. Please reopen if you have more details.

**Comment by yuping3252** [comment]
I have the same issue. I am installing cuda libraries on Ubuntu 20.04, After installation, I ran a small python program which contains one line:
import tensorflow as tf

This is part of the warning message when  I ran above statement:

2021-04-09 19:52:41.024315: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-04-09 19:52:41.024341: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

The missing library I found is 
/usr/local/cuda-11.2/lib64/libcudart.so.11.0
which is a soft link to dynamic library libcudart.so.11.2.152 in the same directory

I added the path /usr/local/cuda-11.2/lib64/ to my PATH variable, still, when I ran that python , it can't find the library.
I added the path /usr/local/cuda-11.2/lib64/libcudart.so.11.0, still can't find
I added the path /usr/local/cuda-11.2/lib64/libcudart.so.11.2.152, still can't find

Can't figure out what's the problem
